### PR TITLE
docs: Feature Repo, Types and FAQ doc updates

### PR DESCRIPTION
**Pull request by achals**
Signed-off-by: Achal Shah <achals@gmail.com>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

File: docs/getting-started/concepts/data-ingestion.md [hunk 1]
```diff
@@ -77,7 +79,14 @@ materialize_bash = BashOperator(
 
 </details>
 
-### Stream data ingestion
+### Batch Data Schema Inference
```
Comment (felixwang9817) [inline docs/getting-started/concepts/data-ingestion.md:82]: for consistency, `Batch data schema inference`

here and elsewhere

File: docs/getting-started/concepts/feature-repo.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Feature Repository
+
+## Feature Repo
+
+A feature repository is the collection of python files that define entities, feature views and data sources. Feature Repos also have a `feature_store.yaml` file at their root. 
+
+Users can collaborate by making and reviewing changes to feast object definitions (feature views, entities, etc) in the feature repo.
```
Comment (felixwang9817) [inline docs/getting-started/concepts/feature-repo.md:7]: for consistency, `Feast` instead of `feast

**Review by felixwang9817 — COMMENTED** [review]
looks good overall

**Review by felixwang9817 — APPROVED** [review]
/lgtm

File: docs/getting-started/concepts/feature-repo.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Feature Repository
```
Comment (felixwang9817) [inline docs/getting-started/concepts/feature-repo.md:1]: @achals oops just realized we already have a [page](https://docs.feast.dev/getting-started/architecture-and-components/feature-repository) for feature repos

we should make sure to merge this in with that page somehow

Comment (achals) [reply docs/getting-started/concepts/feature-repo.md:1]: I'll merge

**Pull request merged**